modules/tracker/models/base/ScalarModelBase.php

```diff
@@ -117,88 +65,32 @@ public function __construct()
     abstract public function getByTrendAndTimestamp($trendId, $submitTime, $userId = null);
 
     /**
-     * Return all distinct branch names of revisions producing scalars.
+     * Return any other scalars from the same submission as the given scalar.
      *
-     * @return array branch names
+     * @param Tracker_ScalarDao $scalarDao scalar DAO
+     * @return array scalar DAOs
      */
-    abstract public function getDistinctBranches();
+    abstract public function getOtherScalarsFromSubmission($scalarDao);
```

Comment:
Is there a reason to continue supporting this search by time functionality?  Aren't we doing everything with scalars tied to submissions now?